spec/models/manageiq/providers/vmware/infra_manager/orchestration_template_spec.rb

```diff
@@ -1,11 +1,11 @@
 describe ManageIQ::Providers::Vmware::InfraManager::OrchestrationTemplate do
+  let(:ems) { FactoryBot.create(:ems_vmware) }
+  let(:resource_pool) { FactoryBot.create(:resource_pool, :ems_ref => 'obj-103') }
+  let(:options) { {:accept_all_EULA => false, :resource_pool_id => resource_pool.id} }
```

Comment:
@lfu will this need to change after https://github.com/ManageIQ/manageiq-providers-vmware/pull/640/files#diff-36ae7efcb639c86475defc746eba49acR50 ?

Reply:
Depends on which PR got merged first. I will take care of that.